dDatabase/CDClientDatabase/CDClientTables/CDObjectsTable.cpp

```diff
@@ -1,7 +1,7 @@
 #include "CDObjectsTable.h"
 
 namespace {
-	CDObjects m_default;
+	CDObjects objDefault;
```

Comment:
file scope should be PascalCase.  Helps to discern what is a global and what isnt.  only reason this wasnt is because I was editing too many files at that time to warrant variable name changes.

Reply:
Not sure I agree with that formatting decision. Isn't it much more important to preserve the clear distinction between functions and objects than to convey the file scope? One is a very easy to catch compiler error (if not quickly found by the red squiggles in an IDE). The other could introduce very subtle and hard to diagnose bugs if a PascalCase variable was somehow used in a context where a function or object was expected instead.

In particular, I really don't like how your proposed formatting rule would interact with constructor calls. ObjName objName{}; is much clearer than ObjName ObjName{}; If you use parenthesis, the latter might even look like a function call: ObjName ObjName().

Reply:
Resolution agreed upon is f_ for file scope as per discord conversation